jocktos/src/tcb.c

```diff
@@ -1,9 +1,30 @@
+/**
+* \brief This module contains the logic for a single linked list of
+* Task Control Blocks
+*/
+/* -- Includes ------------------------------------------------------------ */
+// Jocktos
 #include "tcb.h"
+// Middleware
+// Bios
+// Standard C
 #include <stdlib.h>
 
+/* -- Defines ------------------------------------------------------------- */
+
+/* -- Types --------------------------------------------------------------- */
+
+/* -- Local Globals (not for libraries with application instantiation) ---- */
+
 T_TCBError JOCKTOS_TCBError = {0, 0, 0, 0, 0};

```

Comment:
I think we can just set the entire structure to 0 just with one of these...

```suggestion
T_TCBError JOCKTOS_TCBError = {0};
```

C99 References this explicitly in that if there are fewer initializers in the list, the remaining members are initialized implicitly the same as objects that have been inited